contextvh/src/main/java/contextvh/ContextEntity.java

```diff
@@ -30,6 +36,7 @@
 	 */
 	public ContextEntity(final EntityListener env, final String intendedStakeholder, final Integer slotID) {
 		super(env, intendedStakeholder, slotID);
+		customActions.addAction(perceptsFilter);
```

Comment:
@MaxGroenenboom It worked a bit differently (with a singleton pattern),but I don't think it would be a big difference since `contextEntity` would still need to be updated when `FilterPercepts` would be removed.